recipes/pyqode.python/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pyqode.python" %}
+{% set version = "2.10.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e03855e70fef8606f378afa3ac41938eab601749bdfd00d922bb3fc7c4dccd81
+
+build:
+  number: 0
+  noarch: python
+  preserve_egg_dir: True
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - docutils
+    - jedi
+    - pep8
+    - pip
+    - pyflakes
+    - pyqode.core
+    - pyqode.qt
+    - python
+    - setuptools
+  run:
+    - docutils
+    - jedi
+    - pep8
+    - pyflakes
+    - pyqode.core
+    - pyqode.qt
+    - python
+    - setuptools
```

Comment:
Do you need ``setuptools`` as run requirements?

Reply:
`pkg_resources` is [imported at runtime](https://github.com/pyQode/pyqode.python/blob/72ecf6a4d5c4060d4adae4743657792151d44560/pyqode/__init__.py#L7) so it seems that `setuptools` is needed.
I have just checked it by creating a new environment with only python (`--no-deps` was necessary to avoid installing setuptools at the same time than python). `setuptools ` is a dependency of `pyqode.qt` and `pyqode.core`, so it is going to be installed anyway, so it should not really matter. Although, in case this dependency is removed from these packages, it would break this one, so maybe better to keep here!